rasa/utils/features.py

```diff
@@ -0,0 +1,105 @@
+from typing import Union, Text, Optional, List, Any
```

Comment:
I think the `utils` package is not a good place for this. That's a key class for our policies and hence should be next to the featurizers

Reply:
it is used in nlu as well, so it is outside core and nlu

Reply:
Yeah, it is also used in nlu. I think at some point we need to rethink the structure as we also wanted to move `tensorflow` out of `utils`. Would make sense to create a separate package for the model related stuff as nlu and core model will share more and more code in the future.

Reply:
Why not start right now and create a package and put `features` in it? We can move the remaining things in separate PRs as it's out of scope.

Reply:
Not sure how we should name the package, any suggestions? I would consider this as non-blocking.

Reply:
A natural name would be `model` but there is already a `model` module (which probably also should rather belong into the `model` package then. Due to the size I'd say it's okay to do this in a follow up PR.